__tests__/app.test.js

```diff
@@ -218,5 +218,55 @@ describe('GET /api/articles', () => {
         });
       })  
     })
+});
+describe('POST /api/articles/:article_id/comments', () => {
+  describe('request tests', () => {
+    test('Responds with status 201 on successful request', () => {
+    const newComment = {
+      username: 'butter_bridge',
+      body: 'This is a test comment!'
+    }
+    return request(app)
+    .post('/api/articles/2/comments')
+    .send(newComment)
+    .expect(201)
+    })
+    test('Responds with newly created object with the correct property labels representing a comment', () => {
+      const newComment = {
+        username: 'butter_bridge',
+        body: 'This is a test comment!'
+      }
+      return request(app)
+      .post('/api/articles/2/comments')
+      .send(newComment)
+      .then(({body}) => {
+        expect(body.comment).toMatchObject({
+          comment_id: expect.any(Number),
+          body: expect.any(String),
+          article_id: expect.any(Number),
```

Comment:
WE could be more explicit here and assert the correct/matching article id here

Reply:
Still needed ^^^